crates/rooch-framework/sources/coin.move

```diff
@@ -736,115 +778,4 @@ module rooch_framework::coin {
         account_storage::global_move_to<Capabilities<CoinType>>(ctx, account, cap)
     }
 
-    //
-    // Tests
-    //
-
-    #[test_only]
-    struct FakeCoin {}
-
-    #[test(source = @0xa0a, destination = @0xb0b, mod_account = @rooch_framework)]
-    public entry fun test_end_to_end(
```

Comment:
Test cases in `coin.move` is all for entry funciton, move them into `coin_test.move`?

Reply:
Done